`buildSearchableOptions`: Execution failed; Index: 1, Size: 1

Recently the build workflow has been failing out of the blue (e.g., [1](https://github.com/InSyncWithFoo/ryecharm/actions/runs/11074230954/job/30772552166), [2](https://github.com/InSyncWithFoo/ryecharm/actions/runs/11080443021/job/30790895809), [3](https://github.com/InSyncWithFoo/ryecharm/actions/runs/11085736415/job/30802466763), [4](https://github.com/InSyncWithFoo/ryecharm/actions/runs/11088081961/job/30807523905), [5](https://github.com/InSyncWithFoo/ryecharm/actions/runs/11088149720/job/30807684738)):

```text
> Task :buildSearchableOptions FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildSearchableOptions'.
> Index: 1, Size: 1
```

It is unknown what the cause is. I temporarily disabled the task in [`54d690d`](https://github.com/InSyncWithFoo/ryecharm/commit/54d690d18e919f44583946172cd84102dae78704) so that other jobs are not blocked. This has the unfortunate consequence of settings not being searchable.

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -30,6 +30,12 @@ jobs:
         name: Fetch sources
         uses: actions/checkout@v4
       -
+        name: Install IJGP patch
+        run: |
+          git clone https://github.com/JetBrains/intellij-platform-gradle-plugin
+          cd intellij-platform-gradle-plugin
+          git reset 7b4e6b7f3dab7e0966ffc8c053a4972b9175058a
+      -
         name: Gradle wrapper validation
         uses: gradle/actions/wrapper-validation@v4
       -
@@ -138,6 +144,12 @@ jobs:
         name: Fetch sources
         uses: actions/checkout@v4
       -
+        name: Install IJGP patch
+        run: |
+          git clone https://github.com/JetBrains/intellij-platform-gradle-plugin
+          cd intellij-platform-gradle-plugin
+          git reset 7b4e6b7f3dab7e0966ffc8c053a4972b9175058a
+      -
         name: Set up Java
         uses: actions/setup-java@v4
         with:
@@ -232,6 +244,12 @@ jobs:
         name: Fetch sources
         uses: actions/checkout@v4
       -
+        name: Install IJGP patch
+        run: |
+          git clone https://github.com/JetBrains/intellij-platform-gradle-plugin
+          cd intellij-platform-gradle-plugin
+          git reset 7b4e6b7f3dab7e0966ffc8c053a4972b9175058a
+      -
         name: Set up Java
         uses: actions/setup-java@v4
         with:
@@ -307,6 +325,12 @@ jobs:
       -
         name: Fetch sources
         uses: actions/checkout@v4
+      -
+        name: Install IJGP patch
+        run: |
+          git clone https://github.com/JetBrains/intellij-platform-gradle-plugin
+          cd intellij-platform-gradle-plugin
+          git reset 7b4e6b7f3dab7e0966ffc8c053a4972b9175058a
       -
         name: Set up Java
         uses: actions/setup-java@v4

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -76,8 +76,6 @@ dependencies {
 
 // Configure IntelliJ Platform Gradle Plugin - read more: https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-extension.html
 intellijPlatform {
-    buildSearchableOptions = false
-    
     pluginConfiguration {
         name = providers.gradleProperty("pluginName")
         version = providers.gradleProperty("pluginVersion")

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -3,3 +3,5 @@ rootProject.name = "ryecharm"
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
 }
+
+includeBuild("intellij-platform-gradle-plugin")

